python/artm/regularizers.py

```diff
@@ -66,29 +67,63 @@ def _update_config(self, obj, first=False):
 
 
 class Regularizers(object):
-    def __init__(self, master):
+    def __init__(self, master, force_add=False):
```

Comment:
In my opinion, `force_add` intended to be global options that affects all assignment operators, including `__setitem__` (e.g. `model.regularizers['somename'] = SomeReg()`). I don't know where to specify `replace=True` or smth like that when using operator[]-like style.